app.json

```diff
@@ -109,7 +113,7 @@
    },
    "buildpacks": [
      {
-       "url": "https://github.com/dmathieu/heroku-buildpack-submodules#b37ffe4361bb9c975dd8e93068c9d296365d748c"
+       "url": "https://github.com/dmathieu/heroku-buildpack-submodules#0caf30af7737bf1bc32b7aafc009f19af3e603c1"
```

Comment:
Why do we need to change the buildpack? Shall we do that on production, too?

Reply:
I'm not sure about the production - Heroku may work differently for already deployed apps and previous version of this particular buildpack will work perfectly fine.
However, for the new builds https://github.com/dmathieu/heroku-buildpack-submodules/blob/master/bin/install_git_submodules.rb throws an error and https://github.com/dmathieu/heroku-buildpack-submodules/commit/35622728df8757a1bacde155290fcd2960ff1a11 seems to fix that.

Reply:
Gotcha!